src/hooks/useCandidateFilterUrlParams.ts

```diff
@@ -1,14 +1,54 @@
-import { useCallback } from 'react';
+import { useCallback, useEffect, useMemo } from 'react';
 import { InterviewStatus } from '@prisma/client';
 import { debounce } from 'throttle-debounce';
 import { useRouter } from 'next/router';
-import { useUrlParams } from '@taskany/bricks';
+import { deleteCookie, setCookie, useUrlParams } from '@taskany/bricks';
 
+import { trpc } from '../trpc/trpcClient';
+import { filtersNoSearchPresetCookie } from '../utils/filters';
 import { GetCandidateList } from '../modules/candidateTypes';
 
-export const useCandidateFilterUrlParams = () => {
+export const useCandidateFilterUrlParams = ({
+    defaultPresetFallback = false,
+}: { defaultPresetFallback?: boolean } | undefined = {}) => {
     const router = useRouter();
-    const pushUrl = useCallback((url: string) => router.push(url), [router]);
+    const pushUrl = useCallback(
+        (url: string) => {
+            const query = url.split('?')[1];
+
+            if (!query) {
+                setCookie(filtersNoSearchPresetCookie, true, {
+                    'max-age': 30,
```

Comment:
30 seconds ~to Mars~?

Reply:
wee need this cookie only for 1 next render and if something be wrong - it will be removed after 30 second timeout